packages/components/pie-modal/README.md

```diff
@@ -56,12 +57,12 @@ import { PieModal } from '@justeattakeaway/pie-modal/dist/react';
 
 ## Props
 
-| Property       | Type      | Default | Description                                           |
-|----------------|-----------|---------|-------------------------------------------------------|
-| isOpen         | `Boolean` | `false` | Controls if the modal element is open or closed       |
-| heading*       | `String`  | -       | Sets the heading of the modal                         |
-| headingLevel   | `String`  | `h2`    | Allows you to set the heading tag (from `h1` to `h6`) |
-
+| Property      | Type      | Default | Description                                           |
+|---------------|-----------|---------|-------------------------------------------------------|
+| isOpen        | `Boolean` | `false` | Controls if the modal element is open or closed       |
+| heading*      | `String`  | -       | Sets the heading of the modal                         |
+| headingLevel  | `String`  | `h2`    | Allows you to set the heading tag (from `h1` to `h6`) |
+| isDismissible | `Boolean` | `false` | Controls if the modal element is dismissible or not.  |
```

Comment:
nit: I just took a look at react.pie.design, snacks and fozzie and it seems the default value for the modal to show the close button, is there a reason we aren't doing that here?

Reply:
I'm happy for the default to change to `true` if we think that's the most common usecase – I would check with @RLumleyJET in terms of what they consider the most used case and go for that.